### PR TITLE
Feature/downstream ci

### PR DESCRIPTION
**Pull request by figi44**
Add downstream CI which runs tests on platform builders and HPC.

### CODE REVIEW

**Comment by FussyDuck** [comment]
[CLA assistant check](https://bol-claassistant.ecmwf.int:443/ecmwf/pyfdb?pullRequest=7) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://bol-claassistant.ecmwf.int:443/ecmwf/pyfdb?pullRequest=7) before we can accept your contribution.<br/>**2** out of **4** committers have signed the CLA.<br/><br/>:white_check_mark: tlmquintino<br/>:white_check_mark: kinow<br/>:x: oiffrig<br/>:x: figi44<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bol-claassistant.ecmwf.int:443/check/ecmwf/pyfdb?pullRequest=7) it.</sub>

File: tests/test_pyfdb.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        "date": "20191110",
+        "class": "rd",
+        "levelist": "300",
+        "param": "138",
+        "time": "0000",
+        "type": "an",
+    }
+
+    filename = os.path.join(os.path.dirname(__file__), "x138-300bis.grib")
+    print("")
```
Comment (simondsmart) [inline tests/test_pyfdb.py:125]: These tests seem to be very ... noisy/chatty

File: tests/test_pyfdb.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    grib.dump()
+
+    request["levelist"] = [300, "400"]
+    request["expver"] = "xxxx"
+    filename = os.path.join(os.path.dirname(__file__), "foo.grib")
+
+    print("")
+    print("save to file ", filename)
+    with open(filename, "wb") as o, fdb.retrieve(request) as i:
+        shutil.copyfileobj(i, o)
```
Comment (simondsmart) [inline tests/test_pyfdb.py:190]: In this section I don't see the _test_. Only lots of code that (if it doesn't fail) the test will pass.

Do we verify that any of the actions have occurred correctly?

**Review by simondsmart — APPROVED** [review]

File: pyfdb/pyfdb.py [hunk 3]
```diff
@@ -12,14 +12,13 @@
...
-from collections.abc import Iterable
+import os
 
 import cffi
-import os
-from pkg_resources import parse_version
 import findlibs
+from pkg_resources import parse_version
 
-__version__ = '0.0.1'
```
[the remark this answers is not in the document]
Comment (oiffrig) [reply pyfdb/pyfdb.py:22]: Also, there is another `__version__` defined in `pyfdb/version.py`, which is the one used in `setup.py`.

File: tests/test_pyfdb.py (hunk 2 above)
Comment (tbkr) [reply tests/test_pyfdb.py:190]: This still is the initial test.

File: tests/test_pyfdb.py (hunk 1 above)
Comment (tbkr) [reply tests/test_pyfdb.py:125]: This still is the initial test.

**Pull request merged**